tests/js/client/active-failover/basic.js

```diff
@@ -293,10 +296,136 @@ function waitUntilHealthStatusIs(isHealthy, isFailed) {
   return false;
 }
 
+function loadFoxxIntoZip(path) {
+  let zip = utils.zipDirectory(path);
+  let content = fs.readFileSync(zip);
+  fs.remove(zip);
+  return {
+    type: 'inlinezip',
+    buffer: content
+  };
+}
+function checkFoxxService() {
+  const onlyJson = {
+    'accept': 'application/json',
+    'accept-content-type': 'application/json'
+  };
+  let reply;
+  db._useDatabase("_system");
+
+  [
+    '/_db/_system/_admin/aardvark/index.html',
+    '/_db/_system/itz/index'
+    //, '/_db/_system/crud/xxx'
+  ].forEach(route => {
+    for (let i=0; i < 200; i++) {
+      try {
+        reply = arango.GET_RAW(route, onlyJson);
+        if (reply.code === 200) {
+          print(route + " OK");
+          return;
+        }
+        let msg = JSON.stringify(reply);
+        if (reply.hasOwnProperty('parsedBody')) {
+          msg = " '" + reply.parsedBody.errorNum + "' - " + reply.parsedBody.errorMessage;
+        }
+        print(route + " Not yet ready, retrying: " + msg);
+      } catch (e) {
+        print(route + " Caught - need to retry. " + JSON.stringify(e));
+      }
+      internal.sleep(3);
+    }
+    throw ("foxx route '" + route + "' not ready on time!");
+  });
+
+  print("Foxx: Itzpapalotl getting the root of the gods");
+  reply = arango.GET_RAW('/_db/_system/itz');
+  assertEqual(reply.code, "307", JSON.stringify(reply));
+
+  print('Foxx: Itzpapalotl getting index html with list of gods');
+  reply = arango.GET_RAW('/_db/_system/itz/index');
+  assertEqual(reply.code, "200", JSON.stringify(reply));
+
+  print("Foxx: Itzpapalotl summoning Chalchihuitlicue");
+  reply = arango.GET_RAW('/_db/_system/itz/Chalchihuitlicue/summon', onlyJson);
+  assertEqual(reply.code, "200", JSON.stringify(reply));
+  let parsedBody = JSON.parse(reply.body);
+  assertEqual(parsedBody.name, "Chalchihuitlicue");
+  assertTrue(parsedBody.summoned);
+
+ //print("Foxx: crud testing get xxx");
```

Comment:
This is a new test, but the code is commented out.
Can the comments be removed or does the test need to be extended?

Reply:
I copied the complete test from checkdata, which also installs a foxx service that has crud capabilities.
Add service too? Remove? pls decide.

Reply:
If the decision is about whether to remove the commented out or enable it, I'd say let's enable it. Provided it is no major effort to get the extra test code to work.
If it turns out to be a major issue, I suggest removing the commented out test code.

Reply:
it seems more cleanup was needed, so it took a little longer.